pkgs/by-name/sn/snmpwn/package.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, bundlerEnv
+,
+}:
+let
+  rubyEnv = bundlerEnv {
+    name = "snmpwn-ruby-env";
+    gemdir = ./.;
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "snmpwn";
+  version = "0.97b";
+  src = builtins.fetchGit {
+    url = "https://github.com/hatlord/snmpwn";
+    rev = "0dee1d02e1d24159664ad56533a081f0039a68bf";
+  };
+
+  buildInputs = [ rubyEnv.wrappedRuby ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp *.rb $out/bin
+    mv $out/bin/snmpwn.rb $out/bin/snmpwn
+  '';
+
+  meta = with lib; {
+    description = "SNMPv3 user enumerator and attack tool";
+    homepage = "https://github.com/hatlord/snmpwn";
+    license = licenses.mit;
+    maintainers = with maintainers; [ bycEEE ];
```

Comment:
You also need to add yourself to `maintainers/maintainer-list.nix` :)

Reply:
I added myself to `maintainers/maintainer-list.nix` in this recent open PR https://github.com/NixOS/nixpkgs/pull/265083. 

Should I wait for that to be merged in, or add myself to to the maintainer list file in this PR.

Reply:
That PR has two approvals, it should get merged soon. If it doesn't get merged in a few days, you can post it in https://discourse.nixos.org/t/prs-already-reviewed/2617